# max rate of the planner
RATE: 25
REPLAN_RATE: 1000
# goal tolerance
XY_GOAL_TOLERANCE: 0.04
YAW_GOAL_TOLERANCE: 0.15
# maximum speed
V_MAX: 0.25 #0.35
W_MAX: 1.5
COLLISION_AVOID_MAX: 0.1
# acceleration at beginning of movements
ACCELERATION: 1.0
# goal distance for deceleration
D_DECELERATION: 0.15 #0.25
# power of deceleration factor
GAMMA: 0.75
# params for accurate movements to water towers or cube heaps
XY_ACCURATE_GOAL_TOLERANCE: 0.005
YAW_ACCURATE_GOAL_TOLERANCE: 0.05
D_ACCURATE_DECELERATION: 0.25
NUM_RANGEFINDERS: 8
RANGEFINDER_ANGLES: [1.57, 2.36, 3.14, 3.93, 4.71, 5.5, 0.0, 0.785]
RANGEFINDER_DISTANCES: [.055,.11,.115,.155,.14,.155,.12,.115]
COLLISION_STOP_DISTANCE: 150
COLLISION_AVOIDANCE_COEFFICIENT: 0.33
COLLISION_STOP_NEIGHBOUR_DISTANCE: 90
COLLISION_GAMMA: 2
LIDAR_COLLISION_STOP_DISTANCE: 300
MAX_LIDAR_DIST: 400
# params for RRT*
MAX_RRT_ITERS: 1000
STEP: .095
STEP_MIN: .05
STEP_MAX: .25
GAMMA_RRT: 5
EPSILON_GOAL: .05
PATH_WIDTH: .13
PATH_WIDTH_SMALL: .13
PATH_WIDTH_LARGE: .15
PATH_WIDTH_MAX: .15
GOAL_SAMPLE_RATE: 10
CIRCLE_REPLAN_RATE: 100
#params for pure pursuit
LOOKAHEAD: 0.05
DESIRED_DIRECTIONS_OF_TRAVEL: [0.785, 2.36, -2.36, -0.785]